### ISSUE TITLE
Portal ignores destination

### ISSUE THREAD

**Issue by xManInWhitex** [Multiverse/Multiverse-Portals#523]
No matter which destination I choose for a portal (anchor, portal, world spawn, coordinates), MVP will teleport me to a faraway location on the map instead of the specified destination. Teleportation on a single map works. The bug only happens if I want to teleport between worlds.

There is no error in the console or debug file.

Server Version:
Spigot 1.14.4

Multiverse:
Core-4.1.1
Portal-4.1.1
Inventorys-3.0.1

![63809081-b2a87900-c921-11e9-9224-f7f305601c06](https://user-images.githubusercontent.com/54602589/63828024-739c1700-c965-11e9-801a-f70af2b64804.gif)

Plugin List:
ProtocolLib, Multiverse-Core, Multiverse-Portals, Multiverse-Inventories

**Comment by xManInWhitex** [comment]
It is a plugin conflict with ProtocolLib. No matter which ProtocolLib build.

**Comment by xManInWhitex** [comment]
[LogConfigMVProtocolLib.zip](https://github.com/Multiverse/Multiverse-Portals/files/3554040/LogConfigMVProtocolLib.zip)

Here are some Configs and Console Logs with and without PLib

Tested only with MV and Plib no other plugins

**Comment by xManInWhitex** [comment]
Yesterday I'm tested the new build of portals (808) without success.
Also, I closed the ticket on the PLib page because my server is now
running without PLib and Portals is not working yet.
On a server without plugins PLib had caused the problem, 
but the problem seems to be caused by several plugins. 

Excuse me for responding so late, I wanted to wait for a new build of MV-Portals.
My players are currently teleporting with signs. Not so nice.

Do you have an idea?

**Comment by xManInWhitex** [comment]
I will keep testing if I have time.
If I find the reason for this behavior, I'll contact you.

**Comment by xManInWhitex** [comment]
I have come a step further!

The bug only occurs if the portal exists in a nether_portal block.
When I create the portal in air, it works as usual.

Does anybody have an idea?

**Comment by xManInWhitex** [comment]
Okay, just a compilation.

- The bug started with spigot version 1.14.
- The bug still exist in paper version 1.15.
- The bug also exists if only mv-core, mv-portals and mv-inventorys are installed.
- The bug only occurs if the portal is in a **nether_portal** block.

- I've only tested mv-core version 4.1.1, mv-portals version 4.1.1 and mv-inventorys version 3.0.1.
  (lower versions won't work with 1.14.)

After all this knowledge, I think it can only be due to an incompatibility between the map files and multiverse. Otherwise, I can not explain why the bug occurs without other plugins.

**Comment by SiuanSong** [comment]
I believe I am experiencing something along these lines. 
I have my worldborder set to 25k in the overworld and 15k in the nether, which means any portal created in the nether beyond 3125 would return players to the overworld spawn and they'd receive the "You have reached the Edge of this world" message.
My Nether and overworld have 2 different seeds so I linked them via MV Portals. 
I have discovered that any portal in the nether between roughly 2650 and 3125 returns the Edge of the world error. Even though 2,650*8=21,200 and that's well within the worldborder limit of 25k. All portals created between 0 and 2650 in the Nether work perfectly normal.
